src/CabanaPD_Types.hpp

```diff
@@ -14,19 +14,49 @@
 
 namespace CabanaPD
 {
+// Mechanics types.
 struct Elastic
 {
 };
 struct Fracture
 {
 };
+
+// Thermal types.
+struct TemperatureIndependent
+{
+};
 struct TemperatureDependent
 {
 };
-struct TemperatureIndependent
+struct DynamicTemperature : public TemperatureDependent
+{
+    using base_type = TemperatureDependent;
+};
+
+//! Static type checkers.
```

Comment:
Do we need the exclamation mark "!"?

Reply:
That's for automated documentation, but that's not fully set up yet